docs/variants/protectli_vp46xx/test-matrix.md

```diff
@@ -17,10 +17,10 @@ for all platforms of the VP46XX family.
 | 5.   | [NVMe support][NVM]                   | NVM           | All                                  |
 | 6.   | [Custom logo][CLG]                    | CLG           | All                                  |
 | 7.   | [Custom boot menu key][CBK]           | CBK           | All                                  |
-| 8.   | [USB HID and MSC Support][USB]        | USB           | All                                  |
+| 8.   | [USB HID and MSC Support][USB]        | USB           | USB001.XXX and USB002.XXX            |
```

Comment:
For what reason do we omit the other tests from this module?

Reply:
We do not release the following tests on protectli:
- USB003.001 Upload 1GB file on USB storage (Ubuntu 22.04)
- USB003.002 Upload 1GB file on USB storage (Windows 11)